src/coord/src/coord.rs

```diff
@@ -2933,7 +3022,14 @@ impl Coordinator {
     /// `source_id`.
     ///
     /// Updates greater or equal to this frontier will be produced.
-    fn determine_frontier(&mut self, source_id: GlobalId) -> Antichain<Timestamp> {
+    ///
+    /// If `timeline` is specified, the returned timestamp will advance or return
+    /// the `linestamps` linearizability timestamp.
```

Comment:
I don't know what `linestamps` is.

Reply:
Old name from a previous iteration. Fixed, and reworked this entire comment.